README.md

```diff
@@ -71,15 +71,19 @@ apos create-module <module name>
 
 Remember to register the module in `apps.js` with the other module types.
 
+## CLI analytics
+
+To help us better understand how devs are using the CLI, you will have the option to *opt-in* to share basic usage information with the maintainers. This is limited to 1) the command you have executed, 2) if that command succeeded, and 3) a random, anonymous user ID.
```

Comment:
Whoah, that sounds like they can tell when people run apostrophe command line tasks like creating users and groups. I think that impression is scary to give (and we need to make sure it's not true, too).

Honestly I think we should dial back our ambition here and collect solely when a site is created, so we can say "that's all we gather, and we do NOT record the name of the site, so don't worry."

Reply:
I can definitely be more clear. We are intending to get some data on `create-piece`, `create-widget`, and `create-module` as well. But only the CLI-specific stuff. Honestly didn't we come the conclusion that the CLI wasn't super useful for other apostrophe tasks?